src/DetectorMap.cc

```diff
@@ -48,18 +48,6 @@ DetectorMap::DetectorMap(
                                                           wavelengthKnots[ii].end());
         float const maxWavelengthKnot = *std::max_element(wavelengthKnots[ii].begin(),
                                                           wavelengthKnots[ii].end());
-        if (minCenterKnot < bbox.getMinY() || maxCenterKnot > bbox.getMaxY()) {
```

Comment:
When do we need to allow this?

Reply:
The Simulator produces knots that lie off the detector. I could imagine that if they are not included, the solution around the edge could be bad.